### ISSUE TITLE
How to get validation accuracy per epcoh in Yolov8 Classification?

### ISSUE THREAD

**Issue by snehitvaddi** [ultralytics/ultralytics#14577]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How to get validation accuracy per epcoh in Yolov8 Classification?

### Additional

Hello there!
I am working on an image classification problem and after training, I am plotting the metrics. I see that the results.csv has the columns metrics/accuracy_top1, train/loss, val/loss. But I am missing the key metric val/accuracy like the metrics/accuracy_top1.
Is there a way to get the validation accuracy of the model training per epoch?

I know there is an option for "model = val" but this is after the whole training, but I need it for each epoch.

If someone has a workaround or solution, please help.

**Comment by Y-T-G** [comment]
metrics/accuracy_top1 is on validation set

**Comment by Burhan-Q** [comment]
Check the `runs/classify` directory for the training you're looking to get these metrics for. The `results.png` plots use the data saved in `results.csv` and will have these on a per epoch basis.

**Comment by snehitvaddi** [comment]
Thanks for the reply @Y-T-G
Are you sure the "metrics/accuracy_top1" under train is validation accuracy? If so, what is training accuracy then?

Thanks @Burhan-Q and Yes, I did check that plot but there is not label specifying the validation accuracy.
Also, the metrics/accuracy_top5 seems to be a single line. What does that actually mean?
![image](https://github.com/user-attachments/assets/40c094f2-1912-49f4-88ed-cbb076a6a380)

**Comment by Burhan-Q** [comment]
@snehitvaddi `metrics/accuracy_top1` and `metrics/accuracy_top5` are _only_ provided from running validation. They are metrics for accuracy where it measures the highest confidence model predicted class is the same as the ground-truth class (top-1 accuracy) or the measure for the predicted model class is in the five highest confidence values (top-5). These metrics are _only_ possible to generate when comparing against ground truth labels.

**Comment by Y-T-G** [comment]
The training accuracy is not logged.
